### PR TITLE
docs(sass-variables): add nuxt install content

### PR DESCRIPTION
**Pull request by AGDholo**
## Description

Add nuxt install content for sass-variables.md

## Motivation and Context

No one wrote, I did

## How Has This Been Tested?

Vision

## Markup:

Vision

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/docs/src/pages/en/features/sass-variables.md [hunk 1]
```diff
@@ -29,7 +29,30 @@ When you run yarn serve, the vuetify-cli-plugin will automatically hoist the glo
 
 ## Nuxt install
 
-Coming soon. If you are interested in collaborating on this section, please reach out to _johnleider_ in the [community](https://community.vuetifyjs.com).
+This section assumes you have already followed our Nuxt guide on the [Quick start](/getting-started/installation/#nuxt-install) page. The Nuxt Vuetify module exposes an interface to write the configuration. Ensure that you enable the `treeShake` first, this option will use [vuetify-loader](https://github.com/vuetifyjs/vuetify-loader) to enable automatic [treeshaking](/features/treeshaking)(Vuetify SASS variables Only works with treeshaking), enabled only for production by default. Then provide the `customVariables` file path option to custom SASS variables.
```
Comment (nekosaur) [inline packages/docs/src/pages/en/features/sass-variables.md:32]:
```suggestion
This section assumes that you have already followed our Nuxt guide on the [Quick start](/getting-started/installation/#nuxt-install) page. The Nuxt Vuetify module exposes a `vuetify` property where you can configure the module. Ensure that you enable the `treeShake` option first. This option will use [vuetify-loader](https://github.com/vuetifyjs/vuetify-loader) to enable automatic [treeshaking](/features/treeshaking). This is required for custom SASS variables to work. Then provide the `customVariables` file path option to customize SASS variables.
```

File: packages/docs/src/pages/en/features/sass-variables.md [hunk 2]
```diff
@@ -29,7 +29,30 @@ When you run yarn serve, the vuetify-cli-plugin will automatically hoist the glo
...
+}
+```
+
+```scss
+// assets/variables.scss
+
+// Variables you want to modify
+$btn-border-radius: 0px;
+
+// If you need to extend Vuetify SASS lists
```
Comment (nekosaur) [inline packages/docs/src/pages/en/features/sass-variables.md:51]:
```suggestion
// Modifying a SASS map variable
```

**Review by nekosaur — CHANGES_REQUESTED** [review]

File: packages/docs/src/pages/en/features/sass-variables.md [hunk 3]
```diff
@@ -29,7 +29,30 @@ When you run yarn serve, the vuetify-cli-plugin will automatically hoist the glo
...
+```scss
+// assets/variables.scss
+
+// Variables you want to modify
+$btn-border-radius: 0px;
+
+// Modifying a SASS map variable
+$material-light: ( cards: blue );
+
+@import '~vuetify/src/styles/styles.sass';
```
Comment (KaelWD) [inline packages/docs/src/pages/en/features/sass-variables.md:54]: This import isn't needed

Comment (AGDholo) [reply packages/docs/src/pages/en/features/sass-variables.md:54]: It looks like @vuetify/nuxt docs need to be updated

**Review by DRoet — APPROVED** [review]

**Review by KaelWD — APPROVED** [review]

**Pull request merged**